### ISSUE TITLE
Batch norm을 적용하였을 때와 적용하지 않았을 때 차이가 발생하는 이유를 적어주세요

### ISSUE THREAD

**Issue by seokjeongeum** [seungah-Jang/CS182DeepLearning#4]
https://github.com/seungah-Jang/CS182DeepLearning/blob/02a13f5b121dc3a3137313ea2681bf5eabb79b2b/hw1/BatchNormalization.ipynb#L679
